sse2neon.h

```diff
@@ -2023,6 +2023,15 @@ FORCE_INLINE int _mm_movemask_ps(__m128 a)
 #endif
 }
 
+// Check if input 'a' is all 1's, and return 0 if the 'a' is all 1's, otherwise
```

Comment:
Quote from Intel Intrinsic website:
> Compute the bitwise NOT of a and then AND with a 128-bit vector containing all 1's, and return 1 if the result is zero, otherwise return 0.